prisma/readme.md

```diff
@@ -0,0 +1,5 @@
+### DB migration
+
+```
+cargo run --bin prisma -- migrate dev
+```
```

Comment:
good
but you can leave your configuration detailed info

Reply:
yup
no problem